python-threatexchange/version.txt

```diff
@@ -1 +1 @@
-0.0.16
```

Comment:
I don't think this does anything. I don't you can unpush a version.

Reply:
Also, a good example of "Don't bump the version in the same diff that makes changes!"